### DISCUSSION TITLE
引用知网文献无法正确出现URL问题

### DISCUSSION THREAD

**Discussion by lsqzero** [zotero-chinese/wiki#285] [求助 / Q&A] [answered]
大家好，我在应用如下csl文献格式时，无法正确输出URL。
[GB/T 7714—2015（顺序编码，双语，姓名不大写，无 URL、DOI，网络首发文献除外）](https://zotero-chinese.com/styles/gb-t-7714-2015-numeric-bilingual-no-uppercase-no-url-doi-except-aop)
以该文献为例，此时输出结果为
![image](https://github.com/user-attachments/assets/5edf8ba2-6f34-4902-81e4-1440abc03544)
![image](https://github.com/user-attachments/assets/f3c63762-6912-41ef-9008-1d7e26fb666a)
我的水平有限，不知道为什么会出现这样的问题，向大佬们求助。

**Comment by zepinglee** [comment]
导出为 CSL-JSON 格式发上来供我测试一下？

**Reply by lsqzero** [reply]
[Exported Items.json](https://github.com/user-attachments/files/17072421/Exported.Items.json)
您看是不是这个

**Reply by zepinglee** [reply, marked as answer]
测试没有问题呀，有“/OL”载体类型标识和 URL。

<img width="721" alt="Screenshot 2024-09-20 at 17 08 37" src="https://github.com/user-attachments/assets/cd82295c-be2a-49f0-817e-43a409cc1fea">

建议：

1. 检查 CSL 样式是否为最新版；
2. 检查 Zotero 的设置是否去掉了“Include URLs of paper articles in references”。

<img width="587" alt="Screenshot 2024-09-20 at 17 11 02" src="https://github.com/user-attachments/assets/4a40b516-678a-48fc-8d4d-23b173705c44">

**Reply by lsqzero** [reply]
感谢，是我把Include URLs of paper articles in references去掉了。感谢大佬解答疑问。

**Reply by zepinglee** [reply]
奇怪，这里的描述是“only if the article does not have a page range specified”，然而测试文献明明有页码。

**Comment by northword** [comment]
这似乎是预期的？无URL DOI版本，且包含页码

**Reply by zepinglee** [reply]
这个版本是网络首发文献除外，即按照“电子连续出版物中的析出文献”处理，即显示“/OL”载体类型标识和 URL。

这个是按照知网对 GB/T 7714 的理解。
